v2/emailpassword/common-customizations/signin-form/additional-field-props.mdx

```diff
@@ -0,0 +1,164 @@
+
+
+## Setting Default Values
+
+
+To fill in the form fields with preset values in the login widget, add a `getDefaultValue` option to the `formFields` config when initializing SuperTokens on the frontend.
+
+<FrontendSDKTabs>
+<TabItem value="reactjs">
+
+```tsx
+import SuperTokens from "supertokens-auth-react"; 
+import ^{recipeNameCapitalLetters} from "supertokens-auth-react/recipe/^{codeImportRecipeName}";
+import Session from "supertokens-auth-react/recipe/session";
+
+SuperTokens.init({
+    appInfo: {
+        apiDomain: "...",
+        appName: "...",
+        websiteDomain: "..."
+    },
+    recipeList: [
+        ^{recipeNameCapitalLetters}.init({
+            signInAndUpFeature: {
+                signInForm: {
+                    formFields: [{
+                        id: "email",
+                        label: "Your Email",
+                    // highlight-start
+                        getDefaultValue: () => "john.doe@gmail.com"
+                    // highlight-end
+                    }]
+                }
+            }
+        }),
+        Session.init()
+    ]
+});
+```
+</TabItem>
+</FrontendSDKTabs>
+
+<img alt="Prebuilt signin form UI with default values for fields" src="/img/emailpassword/signin-with-default-values.png" />
+
+:::important
+
+The return value of  `getDefaultValue` function must be a string
+
+:::
+
+
+
+## Changing Optional Error Message
+
+When you try to submit login form without a required/non-optional field, the app will throw an error `Field is not optional`.
+You can customize this error message with `nonOptionalErrorMsg` property in the formField config.
+
+Let's see how to achieve it.
+
+<FrontendSDKTabs>
+<TabItem value="reactjs">
+
+```tsx
+import SuperTokens from "supertokens-auth-react";
+import ^{recipeNameCapitalLetters} from "supertokens-auth-react/recipe/^{codeImportRecipeName}";
+import Session from "supertokens-auth-react/recipe/session";
+
+SuperTokens.init({
+    appInfo: {
+        apiDomain: "...",
+        appName: "...",
+        websiteDomain: "..."
+    },
+    recipeList: [
+        ^{recipeNameCapitalLetters}.init({
+            signInAndUpFeature: {
+                signInForm: {
+                    formFields: [{
+                        id: "email",
+                        label: "Your Email",
+                        placeholder: "Email",
+                    // highlight-start
+                        nonOptionalErrorMsg: "Please add your email"
+                    // highlight-end
+                    }]
+                }
+            }
+        }),
+        Session.init()
+    ]
+});
+```
+</TabItem>
+</FrontendSDKTabs>
+
+<img alt="Prebuilt form UI with custom error message" src="/img/emailpassword/signin-with-custom-error-msg.png" />
+
+Observe how the password field showed the standard error message because a custom message wasn't assigned using `nonOptionalErrorMsg` for this field.
```

Comment:
```suggestion
Observe how the password field displays the standard error message because a custom message wasn't assigned using `nonOptionalErrorMsg` for that field.
```